examples/minireview.rs

```diff
@@ -1,6 +1,393 @@
-use anyhow::Result;
+use anyhow::{anyhow, bail, Context, Error, Result};
+use async_trait::async_trait;
+use config::{Environment, File};
+use futures::{
+    future::{self, Either},
+    pin_mut,
+};
+use ipnet::IpNet;
+use review_database::{migrate_data_dir, Database, Store};
+use review_web::{self as web, graphql::AgentManager, CertManager};
+use serde::Deserialize;
+use std::{
+    collections::HashMap,
+    env, fs,
+    net::SocketAddr,
+    path::{Path, PathBuf},
+    process::exit,
+    sync::{Arc, Mutex},
+};
+use tokio::{
+    signal::unix::{signal, SignalKind},
+    sync::Notify,
+};
+use tracing::{error, info, metadata::LevelFilter};
+use tracing_appender::non_blocking::WorkerGuard;
+use tracing_subscriber::{
+    fmt, prelude::__tracing_subscriber_SubscriberExt, util::SubscriberInitExt, EnvFilter, Layer,
+};
+
+struct MiniCertManager {
+    cert: PathBuf,
+    key: PathBuf,
+}
+
+impl MiniCertManager {
+    pub fn new(cert: PathBuf, key: PathBuf) -> Self {
+        Self { cert, key }
+    }
+}
+
+impl CertManager for MiniCertManager {
+    fn cert_path(&self) -> async_graphql::Result<std::path::PathBuf, anyhow::Error> {
+        Ok(self.cert.clone())
+    }
+
+    fn key_path(&self) -> async_graphql::Result<std::path::PathBuf, anyhow::Error> {
+        Ok(self.key.clone())
+    }
+
+    fn update_certificate(
+        &self,
+        _cert: String,
+        _key: String,
+    ) -> async_graphql::Result<Vec<review_web::graphql::ParsedCertificate>, anyhow::Error> {
+        bail!("Not supported")
+    }
+}
+
+pub struct Manager {
```

Comment:
Does this use `db` or `store`? Shouldn't an empty `struct` suffice?

Reply:
Made it empty